Makefile

```diff
@@ -2,20 +2,33 @@
 ifeq ($(OS),Windows_NT)
     SUFFIX = dll
 else
-    SUFFIX = so
+    OS := $(shell uname -s)
+    ifeq ($(OS),Linux)
+        SUFFIX = so
+    else ifeq ($(OS),Darwin)
+        SUFFIX = dylib
+        DEADBEEF_OSX = /Applications/DeaDBeeF.app
+    endif
```

Comment:
Instead of having 2 ifs, it would be better to do one check for macos and fall back to `.so` suffix.

Reply:
Also don't forget to rename to `UNAME_S`.